ApplicationInsights/JavaScriptSDK/ApplicationInsights.ts

```diff
@@ -106,22 +90,24 @@ export class ApplicationInsights implements IAppInsights, IPlugin, IAppInsightsI
         }
     }
 
-    public sendPageViewInternal(name?: string, url?: string, duration?: number, properties?: Object, measurements?: Object) {
-        var pageView = new PageView(name, url, duration, properties, measurements, this.context.operation.id);
-        var data = new Data<PageView>(PageView.dataType, pageView);
-        var envelope = new Envelope(data, PageView.envelopeType);
+    public sendPageViewInternal(pageView: IPageViewTelemetry, properties?: { [key: string]: any }) {
+        let telemetryItem = TelemetryItemCreator.createItem(pageView, PageView.dataType, PageView.envelopeType, properties);
 
-        this.context.track(envelope);
+        this.context.track(telemetryItem);
 
         // reset ajaxes counter
         this._trackAjaxAttempts = 0;
     }
 
     public sendPageViewPerformanceInternal(pageViewPerformance: PageViewPerformance) {
+        // TODO: Commenting out for now as we this package only supports pageViewTelemetry. Added task 
+        // https://mseng.visualstudio.com/AppInsights/_workitems/edit/1310811
+        /*
         var pageViewPerformanceData = new Data<PageViewPerformance>(
             PageViewPerformance.dataType, pageViewPerformance);
         var pageViewPerformanceEnvelope = new Envelope(pageViewPerformanceData, PageViewPerformance.envelopeType);
         this.context.track(pageViewPerformanceEnvelope);
```

Comment:
thanks this is next for discussion. AI only sends this for the first pageview loaded in sdk session.